data/json/mutations/mutations.json

```diff
@@ -6923,5 +6935,83 @@
     "description": "You have fast reflexes, allowing you to dodge attacks and grabs more easily.",
     "starting_trait": true,
     "dodge_modifier": 2
+  },
+  {
+    "type": "mutation",
+    "id": "THRESH_RABBIT",
+    "name": { "str": "Lagomorph" },
+    "points": 1,
+    "description": "You really want to eat some carrots. Welcome to the Lagomorpha family.",
+    "valid": false,
+    "purifiable": false,
+    "threshold": true
+  },
+  {
+    "type": "mutation",
+    "id": "RABBIT_EARS",
+    "name": { "str": "Rabbit Ears" },
+    "points": 1,
+    "visibility": 8,
+    "ugliness": -1,
+    "description": "Your ears have grown into tall, flexible and pretty rabbit ears.  They're are slender and enhance your hearing beyond the normal range.",
+    "valid": false,
+    "types": [ "EARS" ],
+    "category": [ "RABBIT" ],
+    "hearing_modifier": 1.8
+  },
+  {
+    "type": "mutation",
+    "id": "TAIL_RABBIT",
+    "name": { "str": "Rabbit Tail" },
+    "points": 0,
+    "visibility": 1,
+    "ugliness": -1,
+    "description": "Your stubby tail has become fuzzy and more round.  It looks adorable, but doesn't seem to serve another purpose beyond that.",
+    "types": [ "TAIL" ],
+    "cancels": [ "TAIL_STUB" ],
+    "category": [ "RABBIT" ]
+  },
+  {
+    "type": "mutation",
+    "id": "RABBIT_ADAPTATION",
+    "name": { "str": "Lagomorph Adaptation" },
+    "description": "Your fur keeps you covered in winter, maintaining heat and allowing you to better tolerate cold temperatures.",
+    "bodytemp_modifiers": [ -500, 500 ],
```

Comment:
It cools you down if you're overheating?
Why does it works differently than any other fur mutation?

Reply:
Sorry, I think I got lost with that ability because it was first considered to be part of Aftershock, and I kept it as a way to model the custom rabbit fur sprite in the future. Maybe I should remove it for now.

Reply:
```suggestion
    "bodytemp_modifiers": [ 250, 500 ],
```